lib/core/facets.rb

```diff
@@ -1,3 +1,10 @@
+unless $FACETS_LOADED
```

Comment:
However, that means I would have to put this in a shared file being loaded by the different files (since we don't know that the user will `require 'facets`'; they could as well be requiring `facets/foo` - so I cannot just put that in `facets.rb`)

I think I will go the KISS approach and remove the global altogether. It's not a big deal if the user is warned a couple of times in this case methinks.